packages/ag-grid-theme/stories/ag-grid-theme.stories.tsx

```diff
@@ -1,8 +1,7 @@
 import { AgGridReact } from "ag-grid-react";
-import "ag-grid-community/dist/styles/ag-grid.css";
-import "ag-grid-community/dist/styles/ag-theme-material.css";
-import "../uitk-ag-theme.css";
-import "../salt-ag-theme.css";
+import "ag-grid-community/styles/ag-grid.css";
+import "ag-grid-community/styles/ag-theme-material.css";
```

Comment:
We don't need ag grid material style?

Reply:
There are some icons that we don't provide (ie. the grip icon in DragRowExample)

Reply:
How does the icon work currently? Is it from Sass?

We don't really want go backwards being seen as built on top of other theme, unless no other option is there.

Reply:
the icons come from the theme fonts. no sass involved

Reply:
If icon is the main problem, let's ask @pseys on suggested icons. Some were added last time as well, e.g. group